### DISCUSSION TITLE
Add the functionality to plot the PR curve in the COCO metric indicators of mmdet. 给 mmdet 的 coco metric 指标中增加绘制
 PR 曲线功能

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#656] [MMDetection]
# Add the functionality to plot the PR curve in the COCO metric indicators of mmdet. 
 给 mmdet 的 coco metric 指标中增加绘制
 PR 曲线功能 

Add the functionality to plot the PR curve in the COCO metric indicators of mmdet.

给 mmdet 的 coco metric 指标中增加绘制
 PR 曲线功能

## Task Information

**Repository**：MMDetection

**Difficulty**：Medium

**Points**：30

**Estimated Time Consumption**：28 days

## Task Status

**Claimant's Github ID**：SimonGuoNjust

**Claim Time**：2023年10月25日

**PR Link**：https://github.com/open-mmlab/mmdetection/pull/11094

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by SimonGuoNjust** [comment]
Claim the mission

**Comment by SimonGuoNjust** [comment]
PR:https://github.com/open-mmlab/mmdetection/pull/11094
